android/trifle/src/main/java/app/cash/trifle/TrifleApi.kt

```diff
@@ -4,34 +4,33 @@ import app.cash.trifle.validators.CertChainValidatorFactory
 import app.cash.trifle.validators.CertificateValidatorFactory
 import java.util.Date
 
-object TrifleApi {
+class TrifleApi(private val reverseDomain: String) {
```

Comment:
since you have already tried to make everyting similar to iOS, should we change TrifleApi to just Trifle?

Reply:
Not in this PR as it conflicts with the JVM Trifle object